public/contents/5-manage-back-end-services/tours.json

```diff
@@ -104,11 +104,11 @@
       "fileSize": "21.1 KB",
       "fileName": "usestorageservicev9.zip",
       "fid": 17160,
-      "checksum": "720ec0813b35441cfb8e5e86cbd9f9b8dd961de941308cbf1dbe9466111b3337",
+      "checksum": "b1bd278c5b7acd9fea0b80c03065c7563927b4eaf08c4f7b33846ebff9884d9f",
       "url": "https://opensource.hcltechsw.com/volt-mx-tutorials/hikes/tour/use-storage-services",
       "alias": "hikes/tour/use-storage-services",
-      "description": "Create a storage services, populate, and connect the data to the form.",
```

Comment:
@AshokMamidiHcl I just wanted to check if we are creating multiple storage services in this hike. Could you please point me to the ticket.

Reply:
@mandeepalluru  Here is the the ticket [HPHX-13445](https://jira01.hclpnp.com/browse/HPHX-13445) 
 for use storage services hike.